src/Chapter1.hs

```diff
@@ -652,8 +665,16 @@ Implement a function that returns the first digit of a given number.
 You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
+firstDigit :: Int -> Int
+firstDigit n 
+  | acc >= 10 = firstDigit acc
+  | acc == 0 = abs n
+  | otherwise = acc
+ where acc = ((abs n - lastDigit n) `div` 10)
```

Comment:
When you do `div 10`, you don't need to subtract `lastDigit`, `div` already takes care of this, and it doesn't matter for this function :slightly_smiling_face: